README.md

```diff
@@ -254,11 +255,11 @@ is unspecified, the `/v2/entities` endpoint is used.
 Once an `<entity-id>` is known within the context, individual data entities can
 be manipulated using the `/v2/entities/<entity-id>` endpoint.
 
-It is recommended that entity identifiers should be a URN following
+It is recommended that entity identifiers should be URN's following the
 [NGSI-LD guidelines](https://docbox.etsi.org/ISG/CIM/Open/ISG_CIM_NGSI-LD_API_Draft_for_public_review.pdf),
-therefore each `id` is a URN follows a standard format:
-`urn:ngsi-ld:<entity-type>:<entity-id>`. This will mean that every `id` in the
-context data will be unique.
+therefore each `id` as a URN follows a standard format:
+`urn:ngsi-ld:<entity-type>:<entity-id>`. This helps making every `id` in the
```

Comment:
I’d agree with you if following the naming convention was enough to ensure uniqueness of id’s, but unfortunately, in general, it isn’t—e.g. think of concurrency, coordination among processes, etc. So I suppose it’d be more accurate to say sticking to the naming convention “helps” rather than “ensures”?